modules/@apostrophecms/login/index.js

```diff
@@ -348,7 +394,21 @@ module.exports = {
               username: req.user.username,
               email: req.user.email
             }
-          } : {})
+          } : {}),
+          requirements: Object.fromEntries(
+            Object.entries(self.requirements).map(([ name, requirement ]) => {
+              // server-side functions should not be pushed to browser
+              const {
+                // eslint-disable-next-line no-unused-vars
+                verify,
+                // eslint-disable-next-line no-unused-vars
+                props,
+                ...browserRequirement
```

Comment:
Originally everything else was going to be treated as a prop for the component. That changed when I realized we needed support for async, dynamic props and introduced the props function. So I could switch this to a "what do we want" approach instead of a "what do we not want" approach, sure.